geode-web-api/src/main/resources/swagger.properties

```diff
@@ -12,5 +12,4 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 #
-springfox.documentation.swagger.v2.path=/experimental/api-docs
-springfox.documentation.swagger.v1.path=/v1/api-docs
\ No newline at end of file
+springfox.documentation.swagger.v2.path=/v1/api-docs
```

Comment:
Why is the variable name still "v2" since the value has been changed to "v1"?

Reply:
because swagger library is updated, our swagger version is v2.